rootfs/api/models/app.py

```diff
@@ -211,6 +211,17 @@ def create(self, *args, **kwargs):  # noqa
             except KubeException:
                 self._scheduler.ns.create(namespace)
 
+            if settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC != '':
+                quota_name = '{}-quota'.format(namespace)
+                self.log(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                quota_spec = json.loads(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                self.log('creating Quota {} for namespace {}'.format(quota_name, namespace),
+                         level=logging.DEBUG)
+                try:
+                    self._scheduler.quota.get(namespace, quota_name)
```

Comment:
My only problem with this comes not with the use case, but with changing the default quota later. For example:

 - Admin sets default quota
 - Users create applications
 - Admin adds more nodes to the cluster
 - Admin wants to increase the default quota on all existing apps

Right now this would have to be handled outside of Workflow. I don't necessarily enjoy that. Do you have any thoughts on how we could roll that out eventually?

I think the feature is good enough to ship at the moment, just concerned about rolling out updates in the future and what that would look like.